platform/docs/docs/platform/services/data/DisplaySetService.md

```diff
@@ -27,6 +28,20 @@ Then, the same process is used as was originally done to create new display sets
 
 NOTE: Any instances not matched are NOT added to any display set and will not be displayed.
 
+:::::info[Clarification of Terminology]
+
+Display Sets, which are custom to OHIF, are often confused with different DICOM terms, including study, series, and instances. The following are definitions for these terms to alleviate confusion.
+<br></br>
+
+DICOM Terms:
+* **Study**: A collection of series
+* **Series**: A collection of images or objects
+* **Instance**: Single image or object
+
+OHIF Term:
+* **Display Set**: The view shown to a user (including the layout, annotations, and any other data stored with the series)
```

Comment:
This is actually not true.
DisplaySet is also used in dicom standard https://dicom.innolitics.com/ciods/hanging-protocol/hanging-protocol-display/00720200

Basically displaySet is a set of displayable objects. Can be anything. Often series and displaySets are one to one, but sometimes a series has 2 groups of things (localizer + the rest)